src/unfold.js

```diff
@@ -3,9 +3,9 @@ var _curry2 = require('./internal/_curry2');
 
 /**
  * Builds a list from a seed value. Accepts an iterator function, which returns
- * either false to stop iteration or an array of length 2 containing the value
- * to add to the resulting list and the seed to be used in the next call to the
- * iterator function.
+ * either an empty array to stop iteration or an array with an array of length 2,
```

Comment:
Perhaps _a singleton array containing an array_ in place of _an array with an array_?